deploy/helm/trickster/templates/configmap.yaml

```diff
@@ -10,105 +10,191 @@ metadata:
+        # reap_interval_ms defines how long the cache reaper waits between reap cycles. Default is 1000 (1s)
+        # reap_interval_ms = 1000
 
-            # protocol defines the protocol for connecting to redis ('unix' or 'tcp') 'tcp' is default
-            protocol = {{ .Values.cache.redis.protocol | quote }}
+        # compression determines whether the cache should be compressed. default is true
+        # changing the compression setting will leave orphans in your cache for the duration of record_ttl_secs
+        # compression = true
 
+            ### Configuration options when using a Redis Cache
+            # [cache.redis]
+            # protocol defines the protocol for connecting to redis ('unix' or 'tcp') 'tcp' is default
+            # protocol = 'tcp'
             # endpoint defines the fqdn+port or path to a unix socket file for connecting to redis
             # default is 'redis:6379'
-            endpoint = {{ .Values.cache.redis.endpoint | quote }}
-
-            {{- else if eq .Values.cache.type "filesystem" }}
-            # Configuration options when using a Filesystem Cache
-            [cache.filesystem]
+            # endpoint = 'redis:6379'
+            # password provides the redis password
+            # default is empty
+            # password = ''
 
+            ### Configuration options when using a Filesystem Cache
+            # [cache.filesystem]
             # cache_path defines the directory location under which the Trickster cache will be maintained
             # default is '/tmp/trickster'
-            cache_path = {{ .Values.cache.filesystem.path | quote }}
+            # cache_path = '/tmp/trickster'
+
+            # Configuration options when using a bbolt Cache
+            #[cache.bbolt]
 
-            {{- else if eq .Values.cache.type "boltdb" }}
-            # Configuration options when using a BoltDb Cache
-            [cache.boltdb]
             # filename defines the file where the Trickster cache will be maintained
             # default is 'trickster.db'
-            filename = {{ .Values.cache.boltdb.file | quote }}
+            # filename = 'trickster.db'
 
             # bucket defines the name of the BotlDb bucket (similar to a namespace) under which our key value store lives
             # default is 'trickster'
-            bucket = {{ .Values.cache.boltdb.bucket | quote }}
-            {{- end }}
-
-    # Configuration options for mapping Origin(s)
-    [origins]
-
-        # The default origin
-        [origins.default]
-
-            # origin_url defines the URL of the origin. Default is http://prometheus:9090
-            origin_url = {{ .Values.originURL | quote }}
-
-            # api path defines the path of the Prometheus API (usually '/api/v1')
-            api_path = '/api/v1'
+            # bucket = 'trickster'
+
+      {{- range .Values.caches }}
+        {{ printf "[caches.%s]" .name }}
+        type = {{ .type | quote }}
+        record_ttl_secs = {{ .recordTTLSecs }}
```

Comment:
you may want to pass this through the `default` function:
` .recordTTLSecs | default(21600)`

Since this is an object, it is possible someone forgot this key.